test/domUpdates-test.js

```diff
@@ -0,0 +1,251 @@
+
+  it('should display recipe image', () => {
+    chai.spy.on(domUpdates, ['addRecipeImage'], () => {});
+
+    domUpdates.addRecipeImage(recipeData);
+
+    expect(domUpdates.addRecipeImage).to.have.been.called(1);
+    expect(domUpdates.addRecipeImage).to.have.been.called.with(recipeData);
+  })
+
+  it('should display recipe instructions', () => {
+    chai.spy.on(domUpdates, ['displayInstructions'], () => {});
+    let fullRecipeInfo = {}
+
+    domUpdates.displayInstructions(recipeData, fullRecipeInfo);
+
+    expect(domUpdates.displayInstructions).to.have.been.called(1);
+    expect(domUpdates.displayInstructions).to.have.been.called.with(recipeData, fullRecipeInfo);
+  })
+
+  it('should display all recipes', () => {
+    chai.spy.on(domUpdates, ['showAllRecipes'], () => {});
+
+    domUpdates.showAllRecipes(recipeData);
+
+    expect(domUpdates.showAllRecipes).to.have.been.called(1);
+    expect(domUpdates.showAllRecipes).to.have.been.called.with(recipeData);
+  })
+
+  it('should display pantry info', () => {
+    chai.spy.on(domUpdates, ['displayPantryInfo'], () => {});
+
+    domUpdates.displayPantryInfo(currentUser.pantry);
+
+    expect(domUpdates.displayPantryInfo).to.have.been.called(1);
+    expect(domUpdates.displayPantryInfo).to.have.been.called.with(currentUser.pantry);
+  })
+
+  it('should display searched ingredients', () => {
+    chai.spy.on(domUpdates, ['displaySearchedIngredients'], () => {});
+
+    domUpdates.displaySearchedIngredients(ingredientsData);
+
+    expect(domUpdates.displaySearchedIngredients).to.have.been.called(1);
+    expect(domUpdates.displaySearchedIngredients).to.have.been.called.with(ingredientsData);
+  })
+
+  it('should display negative ingredient increment', () => {
+    chai.spy.on(domUpdates, ['subtractIngredientCount'], () => {});
+
+    domUpdates.subtractIngredientCount(mockEvent);
+
+    expect(domUpdates.subtractIngredientCount).to.have.been.called(1);
+    expect(domUpdates.subtractIngredientCount).to.have.been.called.with(mockEvent);
+  })
+
+  it('should display positive ingredient increment', () => {
+    chai.spy.on(domUpdates, ['addIngredientCount'], () => {});
+
+    domUpdates.addIngredientCount(mockEvent);
+
+    expect(domUpdates.addIngredientCount).to.have.been.called(1);
+    expect(domUpdates.addIngredientCount).to.have.been.called.with(mockEvent);
+  })
+
+  it('should display favorited icon status', () => {
+    chai.spy.on(domUpdates, ['addToMyRecipes'], () => {});
+
+    domUpdates.addToMyRecipes(mockEvent, currentUser, recipeData, ingredientsData);
+
+    expect(domUpdates.addToMyRecipes).to.have.been.called(1);
+    expect(domUpdates.addToMyRecipes).to.have.been.called.with(mockEvent, currentUser, recipeData, ingredientsData);
+  })
+
+  it('should hide unchecked recipes', () => {
+    chai.spy.on(domUpdates, ['hideUncheckedRecipe'], () => {});
+
+    domUpdates.hideUncheckedRecipe(recipeData);
+
+    expect(domUpdates.hideUncheckedRecipe).to.have.been.called(1);
```

Comment:
Going through this, it seems like the spies essentially require the same test-style template. I like that. Makes things much easier! This is great and now we know FOR SURE that our DOM is updating as it should be

Reply:
Such a relief that it's actually more straight forward than I thought!

Reply:
Thanks! I'd still really like to get in and do describe blocks for all of our test methods later on when we do the final polish and linter